mudlet-mapper.xml

```diff
@@ -4300,7 +4300,10 @@ if tmp ~= &quot;&quot; then
 end
 
 local location, markname
-if tonumber(matches[2]) then
+if not matches[3] then
+  markname = matches[2]
+  location = gmcp.Room.Info.num
```

Comment:
Better to use `mmp.currentroom` just in case any refactoring needs to happen later